### ISSUE TITLE
IP address in a csv file is mistakenly identified as a date format in Spreadsheet Editor when create the backup on docker version 7.5.1.1 and restore it on win version 8.0.1.31

### ISSUE THREAD

**Issue by fz0000** [ONLYOFFICE/DocumentServer#2791]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DocumentServer/issues?q=) and did not find an issue describing my bug.

### Operating System of DocumentServer

Windows Server

### Version information

8.0.1.31

### Expected Behavior

The IP address string "10.1.90.10" should keep the original format.

### Actual Behavior

The IP address string "10.1.90.10" is converted to "1/10/1990".

### Reproduction Steps

1. Upload this csv file: [dhcp_static.csv](https://github.com/user-attachments/files/16107037/dhcp_static.csv)
2. Open it in Spreadsheet Editor
3. The preview is correct:
    <img width="322" alt="image" src="https://github.com/ONLYOFFICE/DocumentServer/assets/5061311/dba74e43-a5b9-40d4-95bf-02679630021c">
4.  The IP address is converted to date when click OK: 
    <img width="243" alt="image" src="https://github.com/ONLYOFFICE/DocumentServer/assets/5061311/659c8359-9457-4581-845d-8f9cf6fde0f2">

### Additional information

Hardware:
16 x Intel Xeon Platinum 8163, 32G Memory

Software:
Control panel: 3.5.427
Community Server: 12.6.1280
Document Server: 8.0.1.31
Xmpp Server: 12.0.92

The workspace is installed on a windows server 2022 with `ONLYOFFICE_Workspace_Win-install.latest.exe` (Version: 24.02.802), and then I restored the data from the backup file of a docker version workspace.

I also installed `onlyoffice-documentserver.exe` on another windows server and tried with the integrated example document server, but can not reproduce.

So I'm not sure whether it's an actual defect or due to some unexpected error during my migration action.

**Comment by VladimirProkofiev52** [comment]
Hello! @fz0000 
I've reviewed your issue but wasn't able to replicate it. Could you please let me know the regional settings configured in your O.S. and editor?

Instead of just opening the file directly, try this:

1. Launch the spreadsheet editor.
2. Go to the Data tab.
3. Select Get Data -> From Local TXT/CSV.
4. Choose your file.
Does the issue persist when following these steps?

Additionally, I noticed you're using an outdated version of our editor. We recently released version 8.1.0. Please update to the latest version and check if the problem still exists.

**Comment by fz0000** [comment]
Hi @VladimirProkofiev52 , 

I tried on the old server, but this issue does NOT occur.
The locale and version info of the old server is:
```
docker image version:
onlyoffice/documentserver    7.5.1.1
onlyoffice/communityserver   12.5.2.1848
onlyoffice/controlpanel      3.5.0.516
mysql                        8.0.29
onlyoffice/elasticsearch     7.16.3

locale:
LANG=C.UTF-8
LANGUAGE=
LC_CTYPE="C.UTF-8"
LC_NUMERIC="C.UTF-8"
LC_TIME="C.UTF-8"
LC_COLLATE="C.UTF-8"
LC_MONETARY="C.UTF-8"
LC_MESSAGES="C.UTF-8"
LC_PAPER="C.UTF-8"
LC_NAME="C.UTF-8"
LC_ADDRESS="C.UTF-8"
LC_TELEPHONE="C.UTF-8"
LC_MEASUREMENT="C.UTF-8"
LC_IDENTIFICATION="C.UTF-8"
LC_ALL=
```
---

The region of the new windows server is `Chinese (Simplified, China)`.
The regional settings in the editor is `Chinese` and `中文(中华人民共和国)`.
I also tried to change the region setting of OS and editor to US English but the issue still occurs.

---

I tried your steps and it works well if import the csv file via Data menu:
<img width="233" alt="image" src="https://github.com/ONLYOFFICE/DocumentServer/assets/5061311/5006ffcc-2686-4df2-9237-ec23e256d265">
<img width="147" alt="image" src="https://github.com/ONLYOFFICE/DocumentServer/assets/5061311/c16327f5-0e2a-4125-acf7-ce2865a3bce6">

---
I tried to reinstall the workspace and the problem still exists.
Then I upgrade the DocumentServer to 8.1.0.169 and the format is correct. So it should be already fixed.

I will try again when next workspace bundle is released, but I think it's safe to close this issue as resolved now. Thanks.
